src/content/guides/workspaces.md

```diff
@@ -109,6 +109,46 @@ To use pub workspaces, all your workspace packages (but not your dependencies)
 must have a SDK version constraint of `^3.6.0` or higher.
 :::
 
+<a name='stray-files'></a>
+## Stray files
+
+When you are migrating an existing mono-repo to use Pub workspaces, there will
+be existing "stray" `pubspec.lock` and `.dart_tool/package_config.json` files
+adjacent to each pubspec. These would shadow the `pubspec.lock` and
+`.dart_tool/package_config.json` placed next to the root
+
+Therefore `pub get` will delete any `pubspec.lock` and
+`.dart_tool/package_config.json` located in directories between the root and
+(including) any workspace package.
```

Comment:
```suggestion
`.dart_tool/package_config.json` located in directories between the root,
including any workspace package.
```

Does that keep the same meaning? I think the "and" might be redudant but you can make the call.